### PR TITLE
[R2] Add new tutorial for R2 event notification

### PR DESCRIPTION
**Pull request by harshil1712**
### Summary

Adds new tutorial for R2 event notification

### CODE REVIEW

**Review by kristianfreeman — APPROVED** [review]

File: src/content/docs/r2/tutorials/summarize-pdf.mdx [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+
+import { Render, PackageManagers, Details } from "~/components";
+
+In this tutorial, you will learn how to use [event notifications](/r2/buckets/event-notifications/) to process a PDF file when it is uploaded to an R2 bucket. You will use [Workers AI](/workers-ai/) to summarize the PDF and store the summary as a text file in the same bucket.
+
+## Prerequisites
+
+To continue, you will need:
+
+1. A [Cloudflare account](https://dash.cloudflare.com/sign-up/workers-and-pages) with access to R2.
```
Comment (dcpena) [inline src/content/docs/r2/tutorials/summarize-pdf.mdx]:
```suggestion
- A [Cloudflare account](https://dash.cloudflare.com/sign-up/workers-and-pages) with access to R2.
```

File: src/content/docs/r2/tutorials/summarize-pdf.mdx [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+2. Have an existing R2 bucket. Refer to [Get started tutorial for R2](/r2/get-started/#2-create-a-bucket).
```
Comment (dcpena) [inline src/content/docs/r2/tutorials/summarize-pdf.mdx]:
```suggestion
- Have an existing R2 bucket. Refer to [Get started tutorial for R2](/r2/get-started/#2-create-a-bucket).
```

File: src/content/docs/r2/tutorials/summarize-pdf.mdx [hunk 3]
```diff
@@ -0,0 +1,480 @@
...
+3. Install [`Node.js`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
```
Comment (dcpena) [inline src/content/docs/r2/tutorials/summarize-pdf.mdx]:
```suggestion
- Install [`Node.js`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
```

**Review by dcpena — APPROVED** [review]
Approved with three small suggestions to align with our style guide.

**Pull request merged**